docs/releasing.md

```diff
@@ -13,4 +13,15 @@ and after the automatically generated pull request is approved and merged
 
 	src/scripts/release.sh deploy X.Y.Z
 
-will create and push the tag which will trigger Travis and Appveyor to depoy the new release.
+will create and push the release tag which will then trigger Travis and Appveyor to deploy the artefacts.
+
+The generated [Github Release]((https://github.com/anz-bank/sysl/releases)) is created as a draft and needs to be manually published after adding release notes.
+
+
+Artefact deployment in detail
+-----------------------------
+Travis CI creates and deploys `sysl-X.Y.Z-py2-none-any.whl`, `sysl-X.Y.Z.tar.gz` and `sysl-lib-X.Y.Z.jar` to the [Sysl Github Release page](https://github.com/anz-bank/sysl/releases) and Appveyor CI adds `sysl.exe`.
+
+Travis also deploys Sysl's wheel and sdist distributions to [PyPI](https://pypi.python.org/pypi/sysl) and the Sysl Java library `sysl-lib-X.Y.Z.jar` to [BinTray](https://bintray.com/anz-bank/maven/sysl-lib/).
+
+Sysl generated Java code might have a dependency on `sysl-lib-X.Y.Z.jar`. In order to resole this dependency for instance with gradle, add `compile 'io.sysl:sysl-lib:X.Y.Z'` to your `build.gradle` file (see [sysl-lib on BinTray](https://bintray.com/anz-bank/maven/sysl-lib/) for further details).
```

Comment:
*To resolve this dependency with gradle, add …

Should have maven instructions too. #88

Reply:
Done